packages/nvidia_nat_core/src/nat/front_ends/fastapi/routes/generate.py

```
@@ -0,0 +1,274 @@
+from .common_utils import _build_interactive_runner
+from .common_utils import _with_annotation
+from .common_utils import get_single_endpoint
+from .common_utils import get_streaming_endpoint
+from .common_utils import post_single_endpoint
+from .common_utils import post_streaming_endpoint
+
+logger = logging.getLogger(__name__)
+
+
+def get_streaming_raw_endpoint(*,
+                               worker: Any | None = None,
```

Ruff (0.15.0)

[warning] Unused function argument: `worker`

(ARG001)